From ad6465e21d8a0d092479b9f8578b35fd28a5a5c8 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Sun, 15 Sep 2019 18:46:57 +0200
Subject: [PATCH] Use CC-BY-3.0

It is unclear to AppStream whether MIT is acceptable for metadata, and it gets
rejected on flathub, use Creative Commons instead:
https://creativecommons.org/licenses/by/3.0/

See https://github.com/hughsie/appstream-glib/issues/320
---
 linux/org.qgis.qgis.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux/org.qgis.qgis.appdata.xml.in b/linux/org.qgis.qgis.appdata.xml.in
index 3dd24a2d78b..0570037dd4f 100644
--- a/linux/org.qgis.qgis.appdata.xml.in
+++ b/linux/org.qgis.qgis.appdata.xml.in
@@ -8,7 +8,7 @@
   </screenshots>
   <description><p>QGIS is a user friendly Open Source Geographic Information System (GIS) licensed under the GNU General Public License. QGIS is an official project of the Open Source Geospatial Foundation (OSGeo). It runs on Linux, Unix, Mac OSX, Windows and Android and supports numerous vector, raster, and database formats and functionalities.</p></description>
 
-  <metadata_license>MIT</metadata_license>
+  <metadata_license>CC-BY-3.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <url type="bugtracker">https://github.com/qgis/QGIS/issues</url>
   <url type="donation">https://qgis.org/en/site/getinvolved/donations.html</url>
